backend/src/v2/compiler/argocompiler/container.go

```diff
@@ -328,6 +331,9 @@ func (c *workflowCompiler) addContainerExecutorTemplate(name string, refName str
 	args := []string{
 		"--copy", component.KFPLauncherPath,
 	}
+	if !*c.cacheEnabled {
+		args = append(args, "--cache_enabled", "false")
```

Comment:
Could you add an Argo compiler unit test with the cache disabled?

Reply:
Done.